lib/std/mem/Allocator.zig

```diff
@@ -215,46 +265,92 @@ fn allocWithSizeAndAlignment(self: Allocator, comptime size: usize, comptime ali
 }
 
 fn allocBytesWithAlignment(self: Allocator, comptime alignment: u29, byte_count: usize, return_address: usize) Error![*]align(alignment) u8 {
-    // The Zig Allocator interface is not intended to solve alignments beyond
-    // the minimum OS page size. For these use cases, the caller must use OS
-    // APIs directly.
-    comptime assert(alignment <= mem.page_size);
```

Comment:
Should this not be an assertion for it to be within the bounds of page_size min and max?

Reply:
No, in fact the debug allocator relies on alignments greater than page_size_max